src/hdskins/java/com/voxelmodpack/hdskins/skins/LegacySkinServer.java

```diff
@@ -144,24 +142,11 @@ private static void verifyServerConnection(Session session, String serverId) thr
         service.joinServer(session.getProfile(), session.getToken(), serverId);
     }
 
-    /**
-     * Should be in the format {@code legacy:http://address;http://gateway}. Gateway is optional.
-     */
-    static LegacySkinServer from(String parsed) {
-        Matcher matcher = Pattern.compile("^legacy:(.+?)(?:;(.*))?$").matcher(parsed);
-        if (matcher.find()) {
-            String addr = matcher.group(1);
-            String gate = matcher.group(2);
-            return new LegacySkinServer(addr, gate);
-        }
-        throw new IllegalArgumentException("server format string was not correct");
-    }
-
     @Override
     public String toString() {
-        return MoreObjects.toStringHelper(this)
-                .add("address", address)
-                .add("gateway", gateway)
-                .toString();
+        return new ToStringBuilder(this, IndentedToStringStyle.INSTANCE)
```

Comment:
Why not use @Expose and a Gson?

Reply:
This is for displaying on screen. It looks like this
```
LegacySkinServer
  address=XXX
  gateway=XXXX
```